examples/data-objects/codemirror/src/index.ts

```diff
@@ -46,10 +49,31 @@ class CodeMirrorFactory implements IRuntimeFactory {
 
         return runtime;
     }
+
+    public async instantiateFirstTime(context: IContainerContext): Promise<IRuntime> {
+        const runtime = await this.loadRuntime(context, false);
+        await runtime.createRootDataStore(defaultComponent, defaultComponentId);
+        return runtime;
+    }
+
+    public async instantiateFromExisting(context: IContainerContext): Promise<IRuntime> {
+        const runtime = await this.loadRuntime(context, true);
+        return runtime;
+    }
+
+    private async loadRuntime(context: IContainerContext, existing: boolean): Promise<ContainerRuntime> {
+        const runtime: ContainerRuntime = await ContainerRuntime.loadStateful(
+            context,
+            this.registry,
+            existing,
+            buildRuntimeRequestHandler(
+                defaultRouteRequestHandler(defaultComponentId),
+                rootDataStoreRequestHandler));
```

Comment:
rootDataStoreRequestHandler - is this intentional change? If so, why new path is different from legacy path?

Reply:
Yes, this was intentional, as `innerRequestHandler` was marked as deprecated.

Reply:
It's good change only if sample keeps working after that change. Which is not guaranteed. :)
I'd move it out to make PR only about one topic.

Reply:
That's a fair point. I should minimize the blast radius :)